Adjust supporter message for first-time givers (#8752)
ppy/osu-wiki: wiki/osu!supporter/en.md

After the transaction is completed, there will be a new event in the `Recent Activity` section in the profile of both the giver and the recipient:

> - `{username} has supported osu! - thanks for your generosity!` if it is the first time the user gets osu!supporter or makes a gift.
> - `{username} has once again chosen to support osu! - thanks for your generosity!` if the user has been an osu!supporter or has made a gift before.

Reviewer: ```suggestion
- `{username} has supported osu! - thanks for your generosity!` if it is the first time the user gets osu!supporter or gifts it to someone else.
- `{username} has once again chosen to support osu! - thanks for your generosity!` if the user has been an osu!supporter or gifted it before.
```
